electricitymap/contrib/capacity_parsers/ENTSOE.py

```diff
@@ -69,21 +70,15 @@ def fetch_production_capacity(
         fuel_code = str(
             timeseries.find_all("mktpsrtype")[0].find_all("psrtype")[0].contents[0]
         )
-        end_date = datetime.strptime(
-            timeseries.find_all("end")[0].contents[0], "%Y-%m-%dT%H:00Z"
-        )
-        if end_date.year != target_datetime.year:
-            pass  # query_ENTSOE fetches data for 2 years, so we need to filter out the data for the previous year
-        else:
-            point = timeseries.find_all("point")
-            value = float(point[0].find_all("quantity")[0].contents[0])
-            if ENTSOE_PARAMETER_BY_GROUP[fuel_code] not in capacity_dict:
-                capacity_dict[ENTSOE_PARAMETER_BY_GROUP[fuel_code]] = {
-                    "value": 0,
-                    "datetime": end_date.strftime("%Y-01-01"),
-                    "source": SOURCE,
-                }
-            capacity_dict[ENTSOE_PARAMETER_BY_GROUP[fuel_code]]["value"] += value
+        point = timeseries.find_all("point")
+        value = float(point[0].find_all("quantity")[0].contents[0])
+        if ENTSOE_CODE_TO_EM_MAPPING[fuel_code] not in capacity_dict:
+            capacity_dict[ENTSOE_CODE_TO_EM_MAPPING[fuel_code]] = {
+                "value": 0,
+                "datetime": f"{target_datetime.year}-01-01",
```

Comment:
Why are we hardcoding the month and day here instead of using the returned start data (with hours). They differ by a few hours for the zones which is what messed up PT capacities.